lib/tests/fixtures/ruby/uses_exit_code.rb

```diff
@@ -0,0 +1,7 @@
+Facter.add(:foo) do
+  setcode do
+    cmd = Gem.win_platform? ? 'cmd /k "exit 99"' : 'sh -c "exit 99"'
```

Comment:
I mean, this is Facter, but I think I've used that test before and feel comfortable relying on it.